curriculum/challenges/english/25-front-end-development/quiz-html-accessibility/66ed9026f45ce3ece4053ebb.md

```diff
@@ -17,439 +17,446 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is accessibility?
 
 #### --distractors--
 
-Placeholder distractor 1
+Accessibility focuses on making websites faster to load.
 
 ---
 
-Placeholder distractor 2
+Accessibility is a belief to make websites compatible with older browsers.
 
 ---
 
-Placeholder distractor 3
+Accessibility is a type of coding style.
 
 #### --answer--
 
-Placeholder answer
+Accessibility is a set of standardized practices that ensure online content can be viewed by users with disabilities.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the best practice for creating accessible headings?
```

Comment:
```suggestion
What is a best practice for creating accessible headings?
```

Another nitpick. There are many best practices for making headings more accessible.